### PR TITLE
 kulturemedia bid adapter: rebrand to dxkulture

### PR DESCRIPTION
**Pull request by dani-nova**
## Type of change
- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter 
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [X ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [X ] Other

## Description of change

Subject: Rebranding: Changing Adapter Name from kulturemedia to dxkulture

Hi Prebid.js Team,

I hope this message finds you well. I'm reaching out to submit a pull request for the rebranding of our adapter from "kulturemedia" to "dxkulture". This change reflects our new branding and aligns our adapter name more closely with company's identity.

Pull Request Details:

From: kulturemedia
To: dxkulture

Changes Made:
Updated all references to the old adapter name "kulturemedia" to the new name "dxkulture" in the codebase.
Adjusted configuration files to use the new name.
Updated any relevant documentation, comments, and variable names to reflect the rebranding.

Motivation for the Change: 
We've recently undergone a rebranding effort, and as part of this process, we're updating our adapter name to "dxkulture". We believe this change will better represent the company and make it easier for users to identify and use our adapter.

Impact Assessment: 
This change has been thoroughly tested internally to ensure that it does not introduce any compatibility issues or regressions. All tests have passed successfully, and the adapter continues to work seamlessly with Prebid.js.

Additional Notes: 
We have taken care to update any relevant documentation in the repository to reflect the new adapter name. Additionally, the rebranding has been communicated to our existing users to avoid any confusion.

We appreciate your attention to this pull request and your consideration of our rebranding efforts. If there are any further changes or adjustments required, please let us know, and we'll be more than happy to address them promptly.

Thank you for your time and ongoing support.
Best regards, 

DXKulture Team

### CODE REVIEW

File: modules/dxkultureBidAdapter.js [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-  logInfo,
-  logMessage
+  isNumber,
+  isArray, logMessage
 } from '../src/utils.js';
 import {registerBidder} from '../src/adapters/bidderFactory.js';
 import {BANNER, VIDEO} from '../src/mediaTypes.js';
 
-const BIDDER_CODE = 'kulturemedia';
+const BIDDER_CODE = 'dxkulture';
```
Comment (patmmccann) [inline modules/dxkultureBidAdapter.js:15]: you may want to add your old name as an alias

**Comment by patmmccann** [comment]
see #10125 for a non-breaking way to rebrand

**Comment by dani-nova** [comment]
> I think I explained why we don't need an alias and I hope that's OK. Could you please explain a bit what is a 'non-breaking' way of rebranding? Reading from your reference... does this mean we would have to wait for a new version of Prebid until our rebranding takes effect? Is there something else we can do to change adapter name without breaking the build process?

**Comment by patmmccann** [comment]
Are you saying you have no publishers at all using your adapter?

**Comment by dani-nova** [comment]
@patmmccann Yes, we don't have any active publishers using our adapter at the moment.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**